### PR TITLE
fix(ui): DOMA-4653 add basic table styles to ui kit mixins

### PR DESCRIPTION
**Pull request by Kekmus**
table in the markdown used to look nothing like the table
Screenshot 2022-12-01 at 08 08 46

now it looks like a table because of the addition of basic table styles into ui kit mixins
Screenshot 2022-12-01 at 08 08 07

### CODE REVIEW

**Review by AleX83Xpert — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

File: packages/ui/src/components/Markdown/style.less [hunk 1]
```diff
@@ -74,4 +75,9 @@
       margin-top: 0.5em;
     }
   }
+
+  table {
+    margin-bottom: 1.8em;
```
Comment (SavelevMatthew) [inline packages/ui/src/components/Markdown/style.less:80]: Why margin bottom to all tables?

File: packages/ui/src/components/style/mixins/table.less [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  min-width: 100%;
+  text-align: left;
+  border: @condo-global-border-width-default solid @condo-global-color-gray-3;
+  border-radius: @condo-global-border-radius-medium;
+  border-spacing: 0;
+
+  th {
+    padding: @condo-global-spacing-20 @condo-global-spacing-16 @condo-global-spacing-16 @condo-global-spacing-16;
+    color: @condo-global-color-gray-7;
+    .condo-typography-text-small();
```
Comment (SavelevMatthew) [inline packages/ui/src/components/style/mixins/table.less:15]: text-small + semiBold = h6

File: packages/ui/src/components/style/mixins/table.less [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+.condo-table {
+  min-width: 100%;
+  text-align: left;
+  border: @condo-global-border-width-default solid @condo-global-color-gray-3;
+  border-radius: @condo-global-border-radius-medium;
+  border-spacing: 0;
+
+  th {
+    padding: @condo-global-spacing-20 @condo-global-spacing-16 @condo-global-spacing-16 @condo-global-spacing-16;
```
Comment (SavelevMatthew) [inline packages/ui/src/components/style/mixins/table.less:13]: Why padding top us from bottom? 
image
```suggestion
    padding: @condo-global-spacing-20 @condo-global-spacing-16;
```

File: packages/ui/src/components/style/mixins/table.less [hunk 4]
```diff
@@ -0,0 +1,29 @@
+@import (reference) "@open-condo/ui/src/tokens/variables.less";
+@import (reference) "@open-condo/ui/src/components/style/mixins/typography";
+@import (reference) "@open-condo/ui/src/components/style/mixins/table";
+
+.condo-table {
+  min-width: 100%;
+  text-align: left;
```
Comment (SavelevMatthew) [inline packages/ui/src/components/style/mixins/table.less:7]: why this rule is important?

File: packages/ui/src/components/Markdown/style.less [hunk 5]
```diff
@@ -74,4 +75,12 @@
       margin-top: 0.5em;
     }
   }
+
+  table {
+    .condo-table();
+  }
+
+  table:not(:last-child) {
+    margin-bottom: 1.8em;
```
Comment (SavelevMatthew) [inline packages/ui/src/components/Markdown/style.less:84]: I guess it's better to use PX here, since it's not font-related

**Pull request merged**